stor/tests/test_dx.py

```diff
@@ -769,6 +844,11 @@ def test_stat_canonical_resource(self):
 
 
 class TestExists(DXTestCase):
+    def test_empty_path(self):
+        dx_p = DXPath('dx://')
+        with pytest.raises(NotImplementedError):
```

Comment:
`NotImplementedError` doesn't make sense. Perhaps this should always return True? Are there other cases where `exists()` causes an error?

Reply:
Not afaik. I don't know if it makes sense to call exists on `dx://`. Sounds like something the user is doing by mistake and should be notified of. Can fix this if you feel strongly about it

Reply:
can imagine it with things like:

```real_paths = [f.exists() for f in paths]``` or something

Reply:
Gotcha. The code snipped above is outdated. This is currently fixed.